### DISCUSSION TITLE
Validate introspection response

### DISCUSSION THREAD

**Discussion by dim-37** [graphql/graphiql#1692] [Q&A / Support]
I tried to validate my `introspection.json`  file with `graphql-inspector` from npm and it returns an error `Error: Unable to read JSON file: FILE_PATH: Syntax Error: Expected <EOF>, found Name "kind".`.
How can I find json schema for graphql schema to validate it or can somebody recommend me a tool to validate introspection?

I don't understand what is wrong whit it :

[introspection.zip](https://github.com/graphql/graphiql/files/5348577/introspection.zip)

**Comment by acao** [comment]
seems like you have an invalid JSON. maybe graphql-js repo can help you?
